### PR TITLE
Airmeet Event Details connector with tests and documentation

### PR DESCRIPTION
**Pull request by strangeways**
This connector adds support for [Airmeet](https://www.airmeet.com/), a webinar platform.

Methods are included for everything listed in the [Event Details API](https://help.airmeet.com/support/solutions/articles/82000909768-1-event-details-airmeet-public-api) documentation. However, we don't currently have data in our account for booths, event tracks, or poll responses, so I haven't verified the data being returned from those.

This connector doesn't implement the [Manage Registrations](https://help.airmeet.com/support/solutions/articles/82000909769-2-manage-registrations-airmeet-public-api) or [Manage Events](https://help.airmeet.com/support/solutions/articles/82000909770-3-manage-event-airmeet-public-api) API methods.

Basic tests and documentation are included.

### CODE REVIEW

**Comment by codygordon** [comment]
@shaunagm since this is a new class, similar to the OpenField connector, do you have any flags? Unlike my PR, looks like Ankur used the APIConnector which is good!

**Review by codygordon — APPROVED** [review]

File: parsons/airmeet/airmeet.py
Comment (shaunagm) [inline parsons/airmeet/airmeet.py:269]: I'm curious about this syntax - does `response["booths"]` return something weird like None?

**Comment by shaunagm** [comment]
Overall this looks good. I am a little concerned about including completely untested code, especially given that this particular API seems to be inconsistent at times in what it returns. Is there any way to add test data so you can double-check?

File: parsons/airmeet/airmeet.py
Comment (strangeways) [reply parsons/airmeet/airmeet.py:269]: @shaunagm Yes, if there are no booths for the Airmeet, the API returns `{"booths":null}`

**Review by shaunagm — CHANGES_REQUESTED** [review]
I'm switching my status to "request changes" because it's my understanding that you're going to test out the last few endpoints in your production environment. Let me know once that's done and I'll approve + merge - or let me know if we need a different approach.

**Comment by shaunagm** [comment]
@strangeways sorry about the failing tests, they're an unrelated issue we're trying to debug and shouldn't block merging (as long as the tests otherwise pass, which they do).

I appreciate the warnings in the docstrings. IIRC that was the only issue that needed addressed before merging, so I will go ahead and merge this

**Pull request merged**
